### ISSUE TITLE
web_protocol.py (ERROR)

### ISSUE THREAD

**Issue by JamesDenby** [jrlucier/eero_tracker#14]
The tracker works, but it is filling my log with errors. Here is an example of what I am seeing:

Error handling request
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/aiohttp/web_protocol.py", line 275, in data_received
    messages, upgraded, tail = self._request_parser.feed_data(data)
  File "aiohttp\_http_parser.pyx", line 523, in aiohttp._http_parser.HttpParser.feed_data
aiohttp.http_exceptions.BadStatusLine: invalid HTTP method

I am on the latest versions (Hass.io and your tracker).

**Comment by jrlucier** [comment]
Thanks for the report! I'm trying to replicate, but I'm not seeing it locally.  Are you sure it's coming from eero_tracker?  The only call out we make is to Eero's API server over HTTPS, and that hasn't changed or is changeable via config.  

The error you're seeing would be applicable if we were calling an HTTPS server over HTTP, or vice-versa. 

`API_ENDPOINT = 'https://api-user.e2ro.com/2.2/{}'`

**Comment by JamesDenby** [comment]
Ok, thank you for responding. I am fairly new to HA so it maybe something else. Here is my configuration.yaml excerpt:

# Device Tracker to track phones through MAC Address
#device_tracker:
#  - platform: eero_tracker
#    consider_home: 300
#    interval_seconds: 60
#    only_macs: "14:c2:13:90:81:77, 30:57:14:58:29:01, c0:a6:00:47:d7:a4, 90:e1:76:62:6a:de"

Since I commented it all out, I no longer get the error, but I will try again and let you know

… James





On 31 Dec, 19, at 04:48, jrlucier <notifications@github.com> wrote:

Thanks for the report! I'm trying to replicate, but I'm not seeing it locally. Are you sure it's coming from eero_tracker? The only call out we make is to Eero's API server over HTTPS, and that hasn't changed or is changeable via config.

The error you're seeing would be applicable if we were calling an HTTPS server over HTTP, or vice-versa.

API_ENDPOINT = 'https://api-user.e2ro.com/2.2/{}'

**Comment by JamesDenby** [comment]
Good news! I reinstalled it and no longer have the error. I guess it was me, something I did, or something else. Thank you for your time and sorry if I wasted it.

… James





On 31 Dec, 19, at 04:48, jrlucier <notifications@github.com> wrote:

Thanks for the report! I'm trying to replicate, but I'm not seeing it locally. Are you sure it's coming from eero_tracker? The only call out we make is to Eero's API server over HTTPS, and that hasn't changed or is changeable via config.

The error you're seeing would be applicable if we were calling an HTTPS server over HTTP, or vice-versa.

API_ENDPOINT = 'https://api-user.e2ro.com/2.2/{}'
